accord-core/src/main/java/accord/messages/Defer.java

```diff
@@ -80,7 +80,7 @@ public void onChange(SafeCommandStore safeStore, SafeCommand safeCommand)
 
         int id = safeStore.commandStore().id();
         if (!waitingOn.contains(id))
-            throw new IllegalStateException();
+            throw new IllegalStateException("Not waiting on CommandStore " + id);
```

Comment:
this is from https://issues.apache.org/jira/browse/CASSANDRA-18364, I hit in burn test so patched here to get stable